test/test-utils/test-utils.ts

```diff
@@ -116,6 +117,10 @@ export function createTestClient(): MatrixClient {
         bootstrapCrossSigning: jest.fn(),
         hasSecretStorageKey: jest.fn(),
 
+        secretStorage: {
+            get: jest.fn(),
+        } as unknown as ServerSideSecretStorage,
```

Comment:
I'm surprised this typecast is necessary: it doesn't seem to be for any of the other properties. Can we get rid of it?

```suggestion
        },
```

Reply:
(also remove the import of `ServerSideSecretStorage`)